readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -17,7 +17,9 @@
 import importlib
 import sys
 import os.path
-from six import string_types
+
+PY3 = sys.version_info[0] == 3
+string_types = str if PY3 else basestring
```

Comment:
We can use a comment here saying that this is copied from `six` and it's to avoid having an extra dependency just for this.